administrator/language/de-DE/plg_workflow_notification.ini

```diff
@@ -4,17 +4,17 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
-COM_WORKFLOW_BASIC_STAGE="Basic Stage"
-COM_WORKFLOW_NOTIFICATION_FIELDSET_LABEL="Notification"
-PLG_WORKFLOW_NOTIFICATION="Workflow - Notification"
-PLG_WORKFLOW_NOTIFICATION_ADDTEXT="The stage has changed"
-PLG_WORKFLOW_NOTIFICATION_ADDTEXT_DESC="This text will be sent: Title [title], changed by [user], new state: [state]. You can add own text to this information. You can localise the text by using a language key and creating a language override."
-PLG_WORKFLOW_NOTIFICATION_ADDTEXT_LABEL="Additional Message Text"
-PLG_WORKFLOW_NOTIFICATION_ON_TRANSITION_MSG="Title: %1$s, changed by %2$s, new state: %3$s."
-PLG_WORKFLOW_NOTIFICATION_ON_TRANSITION_SUBJECT="The status of an %1$s has been changed"
-PLG_WORKFLOW_NOTIFICATION_RECEIVERS_LABEL="Users"
-PLG_WORKFLOW_NOTIFICATION_SENDMAIL_LABEL="Send Notification"
-PLG_WORKFLOW_NOTIFICATION_SENT="Notifications sent"
-PLG_WORKFLOW_NOTIFICATION_USERGROUP_DESC="The users in this usergroup get a notification if this transition has been performed"
-PLG_WORKFLOW_NOTIFICATION_USERGROUP_LABEL="Usergroups"
-PLG_WORKFLOW_NOTIFICATION_XML_DESCRIPTION="Send notification if a transition has been performed in a workflow"
+COM_WORKFLOW_BASIC_STAGE="Grundstufe"
```

Comment:
Ich habe hier zwar "Grundstufe" verwendet, wie bisher in den Screens zu sehen, ich finde aber den Begriff "Eingangsstufe" besser.

Reply:
Die Diskussion hat wir vor einiger Zeit schon und haben uns auf Grundstufe geeinigt.
Wenn das in der Praxis dann anders gewünscht ist, werden wir hoffentlich vor stable erfahren, dann passen wir es an.
Nur sollte dies erstmal einheitlich sein und nicht mit den verschiedensten Übersetzungen gearbeitet werden.

Reply:
OK, dann lasen wir es vorläufig so